src/pages/docs/ai-agents/auto-healing/ai-enabled-auto-healing-for-web-applications.md

```
@@ -0,0 +1,54 @@
+---
+
+---
+
+Testsigma’s AI-Enabled auto-healing automatically detects and fixes broken element locators caused by UI changes during web test execution. This article discusses how auto-healing identifies and updates element locators while executing web test cases.
+
+---
+
+> ## **Prerequisites**
+> 
+> Before you begin, ensure that you have referred to:
+>  1. [Documentation on enabling the auto-healing feature](https://testsigma.com/docs/ai-agents/auto-healing/enable-the-auto-healing-feature/).
```

markdownlint-cli2 (0.18.1)

Multiple spaces after blockquote symbol

(MD027, no-multiple-space-blockquote)